extensions-compose/src/commonMain/kotlin/com/arkivanov/decompose/extensions/compose/stack/animation/predictiveback/PredictiveBackAnimation.kt

```diff
@@ -48,22 +48,23 @@ fun <C : Any, T : Any> predictiveBackAnimation(
 ): StackAnimation<C, T> =
     PredictiveBackAnimation(
         backHandler = backHandler,
-        animation = fallbackAnimation ?: emptyStackAnimation(),
+        animation = fallbackAnimation,
         selector = selector,
         onBack = onBack,
     )
 
 @OptIn(ExperimentalDecomposeApi::class)
 private class PredictiveBackAnimation<C : Any, T : Any>(
     private val backHandler: BackHandler,
-    private val animation: StackAnimation<C, T>,
+    private val animation: StackAnimation<C, T>?,
     private val selector: (BackEvent, exitChild: Child.Created<C, T>, enterChild: Child.Created<C, T>) -> PredictiveBackAnimatable,
     private val onBack: () -> Unit,
 ) : StackAnimation<C, T> {
 
     @Composable
     override fun invoke(stack: ChildStack<C, T>, modifier: Modifier, content: @Composable (child: Child.Created<C, T>) -> Unit) {
         var activeConfigurations: Set<C> by remember { mutableStateOf(emptySet()) }
+        val fallBackAnimationOrDefault = animation ?: LocalStackAnimation.current.invoke()
```

Comment:
Let's apply the same logic here as in `Children`. The variable can be named just `animation`.